Configuration/TestBase.xcconfig

```diff
@@ -7,7 +7,6 @@ LD_RUNPATH_SEARCH_PATHS[sdk=iphone*] = $(inherited) @executable_path/Frameworks
 LD_RUNPATH_SEARCH_PATHS[sdk=appletv*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
 LD_RUNPATH_SEARCH_PATHS[sdk=macosx*] = $(inherited) @executable_path/../Frameworks @loader_path/../Frameworks;
 
-TEST_HOST[sdk=iphone*] = $(BUILT_PRODUCTS_DIR)/TestHost.app/TestHost;
```

Comment:
This makes it impossible to run the tests on device.

Reply:
Are you referring to real devices?, If I remove this swiftui test can be executed both on local and CI.

Reply:
Do we really need to run this test on a actual device now?